### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by Sakura0124** [VAST-AI-Research/TripoSG#49]
Thank you for sharing such an excellent work. Do you have any plans to provide VAE training code or loss function in the near future?
